scripts/linters/general_purpose_linter.py

```diff
@@ -174,6 +174,35 @@
             'extensions/value_generators/',
             'extensions/visualizations/')
     },
+    {
+        'regexp': re.compile(r'tothrow[(]'),
+        'message': 'Please use \'toThrowError\' instead of '
+                   '\'toThrow\'',
+        'excluded_files': (
+            # Note to developers: In the excluded_files below,
+            # we use custom errors which cannot be caught by regex.
+            # These custom errors will be deprecated soon since
+            # they are challenging to maintain.
```

Comment:
Please file an issue and link this to that issue.

Reply:
Done.